org.uqbar.project.xinterpreter/src/org/uqbar/project/wollok/interpreter/api/XInterpreterEvaluator.xtend

```diff
@@ -18,10 +17,11 @@ import org.uqbar.project.wollok.interpreter.core.WollokObject
  * 
  * @author jfernandes
  */
-interface XInterpreterEvaluator extends Serializable {
+interface XInterpreterEvaluator<O> extends Serializable {
```

Comment:
I have abstracted part of the interpreter into a general project called xinterpreter. Therefore the resulto of evaluating depends on the runtime model of the language. In our case WollokObject.